extensions/podman/src/podman-cli.ts

```diff
@@ -16,25 +16,36 @@
  * SPDX-License-Identifier: Apache-2.0
  ***********************************************************************/
 import { spawn } from 'node:child_process';
-
-import type * as extensionApi from '@tmpwip/extension-api';
+import * as extensionApi from '@tmpwip/extension-api';
 import { isMac, isWindows } from './util';
 
 const macosExtraPath = '/usr/local/bin:/opt/homebrew/bin:/opt/local/bin:/opt/podman/bin';
 
 export function getInstallationPath(): string {
   const env = process.env;
+  let path: string = env.Path;
+
+  // Get the path from the configuration if it's been set
+  const containerConfiguration = extensionApi.configuration.getConfiguration('podman');
+  const userPath = containerConfiguration.get('machine.binary');
```

Comment:
should we just return the userPath is defined ?
I'm not sure to see why we need to to the concatenation done later ?